38 Count and Say.js

```diff
@@ -44,11 +44,33 @@ var countAndSay = function(n) {
 //                 count = 1;
 //             }
 //         }
-        
+
 //         newStr += count + str[j - 1];
 //         str = newStr;
 //     }
 
 //     return str;
 // };
 
+
+var countAndSay = function(n) {
+  if(n === 1) {
+    return "1";
+  }
+    var partial = "11";
+
+    for(var i = 3; i <= n; i++) {
+        var newStr = "";
+        for(var j = 0; j < partial.length; j++) {
+            var letter = partial[j];
+            var count = 1;
+            while(partial[j] === partial[j + 1]) {
+                j++;
+                count++;
+            }
+            newStr = newStr + count + letter;
+        }
+        partial = newStr;
```

Comment:
Indent issue.

The idea is the same, but I feel the solution is less clean with the 3 loops inside.